content/articles/php-sessions-and-cookies-to-create-a-secure-remember-me-section-in-a-login-script/Index.md

```diff
@@ -0,0 +1,433 @@
+### Introduction
+
+---
+
+In the Login Script, the 'Remember Me' functionality allows users who have logged in to keep their status.A user's logged-in status is serialized in PHP sessions, cookies, or other similar storage when they select the Remember Me option.
+It's important to be aware of security flaws that could compromise the application's authentication system when storing user login data in a session or a cookie. Cookie storage of plain passwords should be avoided, as this will allow application hacking.
+You will learn how to create a persistent PHP web application by reading this article. When a user attempts to log in to the program, their credentials are validated against the database.
+Before redirecting the user to their dashboard, the PHP session and cookies are used to preserve the user's logged-in state. Session IDs are created when users successfully log in.
+The cookies are then set to save the login name and password for a given length of time. To prevent hacking, a random password and token are produced and kept in the cookie instead of the users' plain password.
+While accessing application pages, a check is made to see if the user is currently logged in, and if not, the page is not displayed. After checking if the session is empty, a cookie check will be performed to determine if the user is logged in. This happens when both the session and cookies do not contain any information about a remembered login.
+With a 1 month expiration date, authentication cookies can be used to authenticate a user. It will be possible to store random passwords and tokens in a database along with their expiration date and time. When logging in, the cookie-based logged-in state validation is performed by checking the database for cookie expiration and availability.
+

```

Comment:
@Juskam, this intro is too wordy. Kindly find a way to remove some content from here and add to the article body. Reason being you have covered almost everything here.